src/QuickMarcEditor/QuickMarcCreateWrapper.js

```diff
@@ -49,6 +50,14 @@ const QuickMarcCreateWrapper = ({
   const [httpError, setHttpError] = useState(null);
 
   const onSubmit = useCallback(async (formValues) => {
+    const controlFieldErrorMessage = checkControlFieldLength(formValues);
+
```

Comment:
Can we place it inside `validateMarcRecord` so we don't have to duplicate this code in each wrapper?

Reply:
@BogdanDenis I thought about it, but it seems that it cannot be done.
With `QuickMarcEditWrapper` everything is ok, but the problem is in the `QuickMarcDuplicateWrapper` and `QuickMarcCreateWrapper`. The `validateMarcRecord` function calls `validateSubfield` which checks if there are empty subfields and if so, an error is returned. So, before calling `validateMarcRecord` we need to call `autopopulateSubfieldSection`, but this function also filters out some field, including 001. Thus, when we finally call `validateMarcRecord`, the `marcRecord` passed to it doesn't contain 001 fields. Visious circle :grimacing: